test/integration/associations/belongs-to-many.test.js

```diff
@@ -1299,7 +1305,7 @@ describe(Support.getTestDialectTeaser('BelongsToMany'), function() {
         }
       });
       this.User.belongsToMany(this.Group, {
-        through: 'group_users',
+        through: 'gu',
```

Comment:
I'm surprised if you only had to change this few table names to get around the long identifier problem.

However, I'd still like to have to solved properly instead of merging a hack, but unfortunately, noone is working on it yet